tests/components/anova/conftest.py

```diff
@@ -78,7 +78,8 @@ async def connect(self) -> None:
         self.ws = MockedAnovaWebsocketStream(self.connect_messages)
         await self.message_listener()
         self.ws = MockedAnovaWebsocketStream(self.post_connect_messages)
-        self.fut = asyncio.ensure_future(self.message_listener())
+        # TODO: fix RUF006 in parent library (AnovaWebsocketHandler)
```

Comment:
We should not add todo's to our codebase. If there is something wrong upstream, let's just denote that and link it up with an external issue.

Reply:
There are a few `TODO` in tests:
- emulated_hue
- energy
- recorder
- zha

I figured that it tests they might have been valid